src/libs/ReportUtils.ts

```diff
@@ -772,8 +772,11 @@ function isReportManager(report: OnyxEntry<Report>): boolean {
 /**
  * Checks if the supplied report has been approved
  */
-function isReportApproved(reportOrID: OnyxInputOrEntry<Report> | string | EmptyObject): boolean {
+function isReportApproved(reportOrID: OnyxInputOrEntry<Report> | string | EmptyObject, parentReportAction: OnyxEntry<ReportAction> = undefined): boolean {
```

Comment:
Why do we need to specifically set the `parentReportAction` default value as `undefined`?

Reply:
@eVoloshchak  Some other places we can use this function without `parentReportAction` param.

Reply:
Where is this used in this PR?